src/main/resources/anniv5Resources/localization/eng/pixiepack/UIstrings.json

```diff
@@ -0,0 +1,14 @@
+{
+  "${ModID}:PixiePack": {
+    "TEXT": [
+      "Pixie",
+      "A pack featuring cards from the dimension hopping Pixie.",
+      "FieryChocobo"
+    ]
+  },
+  "${ModID}:PixiePackModifiers": {
```

Comment:
Is this still used anywhere, or did switching over to BaseMod's version make this no longer needed?

Reply:
It is not, Fixed.